### PR TITLE
CanvasCast: Pinned Items and New Icons

### PR DESCRIPTION
**Pull request by yug2005**
## Description

* You can now pin module items. 
  * You can remove individual recent and pinned items.
  * Move up or down pinned items. 
  * Clear pinned or recent items.
* Updated the Raycast version to 1.38.1
* New Icons.

## Screencast

canvascast 2022-07-22 at 01 54 21

canvascast 2022-07-22 at 01 55 16

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by YodaLightsabr** [comment]
Awesome!

**Comment by danielsequeira** [comment]
@YodaLightsabr did you have a chance to test the improvements and review the code?

**Comment by YodaLightsabr** [comment]
Yeah it looks good!

**Comment by danielsequeira** [comment]
@yug2005 could you add a changelog entry? Then we should be good to merge this one

File: extensions/canvascast/CHANGELOG.md [hunk 1]
```diff
@@ -15,3 +15,9 @@
 - Added modules page
 - Better error handling
 - Added icons
+
+## [Update] - 2022-07-27
+
+- Added pinned module items
+- Added ability to remove recent items
+- Updated icons
```
Comment (danielsequeira) [inline extensions/canvascast/CHANGELOG.md:23]: @yug2005 the most recent entry should be on the top 🙂

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**